### DISCUSSION TITLE
Running in AWS

### DISCUSSION THREAD

**Discussion by aaron-pham** [jdholtz/auto-southwest-check-in#304] [General]
Hey, has anyone tried running this in AWS?

I am thinking of creating some UI with this code as the main logic in the backend and running it on AWS, so my friends and family can easily add flights. I'm unsure though if we'd have issues with SW detecting that we're in AWS and restricting us.

**Comment by dmytrokoren** [comment]
> Hey, has anyone tried running this in AWS?
> 
> 
> 
> I am thinking of creating some UI with this code as the main logic in the backend and running it on AWS, so my friends and family can easily add flights. I'm unsure though if we'd have issues with SW detecting that we're in AWS and restricting us.

That's a cool idea but I think SW is shifting towards assigned seating. I think it will be great effort lost.

Plus, If family flying SW I do their check-in :)
